frontend/packages/ceph-storage-plugin/src/components/create-storage-system/external-storage/ibm-flashsystem/index.tsx

```diff
@@ -166,7 +166,6 @@ export const createFlashSystemPayload: CreatePayload<FlashSystemState> = (
   const storageSecretTemplate: SecretKind = {
     apiVersion: apiVersionForModel(SecretModel),
     stringData: {
-      // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
Hm, I don't see why this comment could be removed. We want the camelcase rule active for most of the code. Did the defaults change?